ceph/ceph_admin/bootstrap.py

```diff
@@ -182,7 +182,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         if custom_repo:
             self.set_tool_repo(repo=custom_repo)
         else:
-            self.set_tool_repo()
+            self.set_cdn_tool_repo()
```

Comment:
I think we are missing out enabling the development build repositories here.
```custom_repo = "ftp://source"
if custom-repo == str: ```
Enable repo using that string will not enable dev build.

Reply:
``` custom_repo
	
	boolean- 
		true: use latest base_url
		false: Enable cdn repo_rpms
	string-
		enable repo with provided string
 ```

Reply:
Thanks @sunilkumarn417 for the logic.